suppressed-cves.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
+    <suppress>
+        <notes><![CDATA[file name: dropwizard-views-1.4.0-SNAPSHOT.jar]]></notes>
+        <gav regex="true">^io\.dropwizard:dropwizard-views(-.*)?:.*$</gav>
+        <cve>CVE-2015-3378</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[file name: dropwizard-views-1.4.0-SNAPSHOT.jar]]></notes>
+        <gav regex="true">^io\.dropwizard:dropwizard-views(-.*)?:.*$</gav>
+        <cve>CVE-2015-3379</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[file name: compiler-0.9.5.jar]]></notes>
+        <gav regex="true">^com\.github\.spullara\.mustache\.java:compiler:.*$</gav>
```

Comment:
I think it's safe to ignore CVE-2015-8862 completely and not just for this artifact.